packages/devextreme/js/ui/list/ui.list.edit.decorator.js

```diff
@@ -82,6 +82,13 @@ const EditDecorator = Class.inherit({
         };
     },
 
+    _updateButtonAttributes($button) {
+        $button.attr({
+            role: null,
+            'aria-label': null,
+        });
```

Comment:
cannot we set tabindex=-1 to exclude the element from the navigation sequence?

Reply:
You are right. It works.